src/modal/modal.js

```diff
@@ -28,6 +28,9 @@ angular.module('mgcrea.ngStrap.modal', ['mgcrea.ngStrap.core', 'mgcrea.ngStrap.h
       var requestAnimationFrame = $window.requestAnimationFrame || $window.setTimeout;
       var bodyElement = angular.element($window.document.body);
 
+      var modalCount = 0;
+      var baseZindexValue = 1037;
```

Comment:
I used the default value for the backdrop set here previously, which was
1038. Then took advantage of the fact that the element is one of the latter
child elements of the body to state that it is on top.

Should I set the modal z-index to start at 1050?
On 5 Nov 2015 23:24, "Olivier Louvignes" <notifications@github.com> wrote:

> In src/modal/modal.js
> <https://github.com/mgcrea/angular-strap/pull/1946#discussion_r44084925>:
>
> > @@ -28,6 +28,9 @@ angular.module('mgcrea.ngStrap.modal', ['mgcrea.ngStrap.core', 'mgcrea.ngStrap.h
> >        var requestAnimationFrame = $window.requestAnimationFrame || $window.setTimeout;
> >        var bodyElement = angular.element($window.document.body);
> >
> > +      var modalCount = 0;
> > +      var baseZindexValue = 1037;
>
> Any reason you picked this specific number? Looks like it should default
> to 1050 according to
> https://github.com/twbs/bootstrap/blob/master/less/variables.less#L277
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mgcrea/angular-strap/pull/1946/files#r44084925>.
>